www-client/seamonkey/seamonkey-2.53.17.1.ebuild

```diff
@@ -75,7 +75,7 @@ BDEPEND="
 		)
 	)
 	virtual/pkgconfig
-	virtual/rust
```

Comment:
You should put a bug reference above BDEPEND explaining why.

Reply:
There have been countless of limitations for dev-lang/rust version of all mozilla products since it's become mandatory, and never ever has there been the need to document a bug reference within the *DEPEND section. I documented everything need in the referenced bug, which isn't solved yet as I haven't had time to contact upstream.

Reply:
I'm asking you to add one. It's useful when something has less frequent bugs to avoid doing a lookup, to see why it's there. Is that really a problem?